_includes/v19.1/known-limitations/system-range-replication.md

```diff
@@ -1 +1 @@
-Changes to the [`.default` cluster-wide replication zone](configure-replication-zones.html#edit-the-default-replication-zone) are not automatically applied to existing replication zones, including pre-configured zones for important system ranges that must remain available for the cluster as a whole to remain available. The zones for these system ranges have an initial replication factor of 5 to make them more resilient to node failure. However, if you increase the `.default` zone's replication factor above 5, consider [increasing the replication factor for important system ranges](configure-replication-zones.html#create-a-replication-zone-for-a-system-range) as well.
+Changes to the [`.default` cluster-wide replication zone](configure-replication-zones.html#edit-the-default-replication-zone) are automatically applied to existing replication zones, including pre-configured zones for important system ranges that must remain available for the cluster as a whole to remain available. The zones for these system ranges have an initial replication factor of 5 to make them more resilient to node failure. However, if you increase the `.default` zone's replication factor above 5, consider [increasing the replication factor for important system ranges](configure-replication-zones.html#create-a-replication-zone-for-a-system-range) as well.
```

Comment:
This is unclear - the second sentence is an exception to the first (and it's not the only one - if you change `range_{min,max}_bytes` or `gc.ttl`, those also won't be applied to the system zones because they have default values for those fields).

Do we really need this callout any more? It was important when even system zones defaulted to 3 replicas. Now that they default to 5, it's saying that if you set your default replication factor to 7 or more you may need to do the same to the system zones, but we don't really recommend running with more than 5 replicas. 

I think there should be a mention on the main page for zone config docs that says that the system zones have their own zone configs that may override settings from `.default`, but we don't need to copy this warning everywhere zone configs are mentioned.